File: Selnium script for testing/Script with Python/test_scenario_30.py
from selenium import webdriver
from selenium.webdriver.common.by import By
import time

def execute_test_30():
    """
    Dummy test run for scenario 30
    """
    driver = webdriver.Chrome()
    driver.implicitly_wait(10)
    driver.get("https://dummy.example.com/page_30")

    driver.find_element(By.CLASS_NAME, "nav-item").click()
    time.sleep(2)
    driver.find_element(By.ID, "user_name").send_keys("test_data_919")
    driver.find_element(By.CLASS_NAME, "nav-item").click()
    driver.find_element(By.CSS_SELECTOR, ".promo-code").send_keys("test_data_581")
    driver.find_element(By.CLASS_NAME, "nav-item").click()
    driver.find_element(By.PARTIAL_LINK_TEXT, "Profile").click()
    assert driver.find_element(By.ID, "success_msg").is_displayed()
    assert driver.find_element(By.ID, "success_msg").is_displayed()
    assert "Dashboard" in driver.title

    driver.quit()

if __name__ == "__main__":
    execute_test_30()

File: Selnium script for testing/Script with Java/src/test/java/testscenario/TestScenario30.java
package testscenario;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;
import java.time.Duration;

public class TestScenario30 {
    public static void executeTest30() {
        /*
         * Dummy test run for scenario 30
         */
        WebDriver driver = new ChromeDriver();
        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
        driver.get("https://dummy.example.com/page_30");

        try {
            driver.findElement(By.cssSelector(".promo-code")).click();
            driver.findElement(By.linkText("Log Out")).sendKeys("test_data_799");
            driver.findElement(By.xpath("//button[@type='submit']")).sendKeys("test_data_159");
            driver.findElement(By.id("checkout_btn")).sendKeys("test_data_852");
            Thread.sleep(2000);
            if (!driver.getTitle().contains("Dashboard")) {
                System.out.println("Dashboard not in title");
            }
            if (!driver.getTitle().contains("Dashboard")) {
                System.out.println("Dashboard not in title");
            }
            driver.findElement(By.name("password_field")).sendKeys("test_data_892");
        } catch (Exception e) {
            e.printStackTrace();
        } finally {
            driver.quit();
        }
    }
    
    public static void main(String[] args) {
        executeTest30();
    }
}
